src/test/java/user/sort/SortAlgorithmsTest.java

```diff
@@ -0,0 +1,110 @@
+package user.sort;
+
+import entity.Review;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+
+public class SortAlgorithmsTest {
+
+    Review review1;
+    Review review2;
+    Review review3;
+    Review review4;
+    Review review5;
+    ArrayList<Review> reviews;
+    ArrayList<Review> correctlySorted;
+
+    @BeforeEach
+    public void setUp(){
+        reviews = new ArrayList<Review>();
+        review1 = new Review("1", "user1", "", 3, "company1", new Date(2022, Calendar.JUNE, 24), 3, 3, new ArrayList<>());
+        review2 = new Review("2", "user2", "", 5, "company1", new Date(2022, Calendar.JULY, 24), 5, 5, new ArrayList<>());
+        review3 = new Review("3", "user4", "", 14, "company1", new Date(2022, Calendar.SEPTEMBER, 24), 115, 115, new ArrayList<>());
+        review4 = new Review("4", "user5", "", -3, "company1", new Date(2022, Calendar.MARCH, 24), 0, 10, new ArrayList<>());
+        reviews.add(review1);
+        reviews.add(review2);
+        reviews.add(review3);
+        reviews.add(review4);
+
+        correctlySorted = new ArrayList<Review>();
+        correctlySorted.add(review3);
+        correctlySorted.add(review2);
+        correctlySorted.add(review1);
+        correctlySorted.add(review4);
+    }
+
```

Comment:
Would it be better to have different resulting correct ArrayLlists of reviews?
(ie. the latest review doesn't necessarily need to be the most helpful one)

Reply:
I already check for differing values of sortable information, and sorting in different ways but getting to the same answer still tests for whether those ways of sorting are working properly. I chose not to make multiple correct sorted lists because it would be redundant and almost double the code length.